syncd/RequestShutdown.h

```diff
@@ -9,6 +9,18 @@
 
 #define SYNCD_NOTIFICATION_CHANNEL_RESTARTQUERY     "RESTARTQUERY"
```

Comment:
I would also remove this since from this point we always want to have specified dB name and probably we can always join dB name to restart query, no need for special case